ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -197,10 +199,10 @@ object Cli {
       .action((_, _) => { println(BuildInfo.Version); sys.exit(0) })
       .text("Prints the version on stdout and exit.")
 
-    opt[Int]("ledger-clock-granularity")
+    opt[Duration]("ledger-clock-granularity")
       .optional()
-      .action((interval, c) => c.copy(ledgerClockGranularityMs = interval))
-      .text("Specify the largest interval in ms that you will see between clock ticks on the ledger under test.  The default is 10000ms")
+      .action((x, c) => c.copy(ledgerClockGranularity = x))
+      .text("Specify the largest interval that you will see between clock ticks on the ledger under test. The default is \"1s\" (1 second).")
```

Comment:
If the documentation does not mention this, it's a good chance to add it, possibly adding a few example and explaining when and how this is useful.

I don't see a problem in adding the new behavior, but I would recommend you also not break current users. This would mean interpreting an absence of unit as `ms`.

Reply:
Good plan. I'll see if I can keep the old behavior as well, but keep it hidden so new users don't find it.

Reply:
Done. Let me know if you're happy with the implementation.

Reply:
LGTM, thanks.